unity/Assets/Maroon/reusablePrefabs/NewRuler/Scripts/Test_RulerDisplay.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Assets.Maroon.reusablePrefabs.NewRuler.Scripts;
+using Maroon.GlobalEntities;
+using Maroon.Physics.CoordinateSystem;
+using TMPro;
+using UnityEngine;
+using Maroon.GlobalEntities;
+
+public class Test_RulerDisplay : MonoBehaviour
+{
+    [SerializeField] private RulerLogic ruler;
+
+    [SerializeField] private TMP_Text DistanceText;
+
+    [SerializeField] private GameObject StartPositionX;
```

Comment:
Can we split the Display in such a way that we have one display for Start and one for End?